stable/jenkins/templates/jenkins-master-deployment.yaml

```diff
@@ -28,6 +28,7 @@ spec:
               {
                   "name": "copy-default-config",
                   "image": "{{.Values.Master.Image}}:{{.Values.Master.ImageTag}}",
+                  "imagePullPolicy": "Always",
```

Comment:
This could probably be "imagePullPolicy": {{.Values.Manifest.imagePullPolicy}}
cc/ @lachie83